### PR TITLE
Fixes #736 (partially) Add relevant comments

### LINKED ISSUE #736: Document all the code files, add relevant comments.
**Actual Behaviour**

The current files need to have comments and logic wherever necessary.

**Would you like to work on the issue?**

@isuruAb @madhavrathi @uday96 and @rishiraj824 can divide the components and work on it.

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #736 

Changes: 

Add comments to following components
- `actions`
- `stores`
- `utils`
- `stores`

### CODE REVIEW

**Comment by rishiraj824** [comment]
@madhavrathi @uday96 Please review

File: src/actions/History.actions.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const cookies = new Cookies();
 
 let ActionTypes = ChatConstants.ActionTypes;
 
-
+// Action to get ueer chat history
```
Comment (mariobehling) [inline src/actions/History.actions.js]: what?

**Comment by mariobehling** [comment]
Comments are ok, but would be great to see more code commits. We have lots of issues opened especially in regards to the skills editor.

**Comment by mariobehling** [comment]
In future please add this while you code and avoid to follow up on this kind of thing later in the process.

**Pull request merged**